must-gather/gather

```diff
@@ -10,14 +13,29 @@ if [ $? -ne 0 ]; then
 fi
 echo "Found subscription ${SUBSCRIPTION_NAME} in namespace ${NAMESPACE}"
 
-/usr/bin/oc adm inspect namespace/${NAMESPACE} --dest-dir=must-gather/
+/usr/bin/oc adm inspect namespace/${NAMESPACE} --dest-dir=must-gather/ &
+pids+=($!)
+echo "PID of adm inspect namespace: ${pids[-1]}"
 
 for CRD in $(/usr/bin/oc get crd | grep secrets-store.csi.x-k8s.io | awk '{print $1}'); do
     echo "Gathering data for CRD ${CRD}"
-    /usr/bin/oc adm inspect ${CRD} --all-namespaces --dest-dir=must-gather/
+    /usr/bin/oc adm inspect ${CRD} --all-namespaces --dest-dir=must-gather/ &
+    pids+=($!)
+    echo "PID of adm inspect ${CRD}: ${pids[-1]}"
 done
 
 echo "Gathering data for ClusterCSIDrivers and CSIDrivers"
-/usr/bin/oc adm inspect clustercsidrivers,csidrivers --dest-dir=must-gather/
+/usr/bin/oc adm inspect clustercsidrivers,csidrivers --dest-dir=must-gather/ &
+pids+=($!)
+echo "PID of adm inspect clustercsidrivers,csidrivers: ${pids[-1]}"
+
+# Check if PID array has any values, if so, wait for them to finish
+# if [ ${#pids[@]} -ne 0 ]; then
+#     echo "Waiting on subprocesses to finish execution."
+#     wait "${pids[@]}"
+# fi
+for pid in "${pids[@]}"; do
+    wait "${pid}" || let RC=$? && echo "job ${pid} with exit code $RC"
+done
 
 exit 0
```

Comment:
RC variable should be used to set actual exit code, now it will just echo a message with the correct code (from RC) and then exit with 0 regardless.